test/extensions/filters/http/buffer/buffer_filter_fuzz.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package test.extensions.filters.http.buffer;
+
+import "envoy/config/filter/http/buffer/v2/buffer.proto";
+import "test/fuzz/common.proto";
+
+import "validate/validate.proto";
+
+message BufferFilterFuzzTestCase {
```

Comment:
Is it possible to use proto reflection and something like `DynamicMessage` with libproto-mutator? This could allow us to synthesize the `HttpData` + per-filter config type message on-the-fly, avoiding the need to have these boiler plate protos.